Marlin/src/lcd/dogm/ultralcd_st7920_u8glib_rrd_AVR.cpp

```diff
@@ -25,7 +25,7 @@
 
 #include "../../inc/MarlinConfig.h"
 
-#if ENABLED(U8GLIB_ST7920) && !defined(U8G_HAL_LINKS) && !defined(__SAM3X8E__)
+#if ENABLED(U8GLIB_ST7920) && !defined(U8G_HAL_LINKS) && !defined(__SAM3X8E__) && !defined(__SAMD51__)
```

Comment:
shouldnt this be if defined(__AVR__) ?

Reply:
This class seems compatible with samd51 like it's for due

Reply:
> shouldnt this be if defined(__AVR__) ?

It looks like the original author intended more than AVR, but we could try it.

Reply:
Apparently Teensy 3.5/3.6 wants this too.

Reply:
Actually, it wants the alternative. Patched.